docs/image_import.md

```diff
@@ -0,0 +1,42 @@
+# Image Import into Google Compute Engine (GCE)
+
+# Introduction
+We provide tooling and instructions for migrating your existing virtual
+machine images to GCE.
+
+Tools:
+* [Daisy](daisy.md)
+* [Import Precheck](../import_precheck/README.md)
+
+## Image Import Process
+The image import process, also known as translation, imports virtual disks to
+GCE disks. Once a disk is imported, additional provisioning and configuration
+is usually needed to make the disk compatible with GCE. After importing,
+provisioning, and configuring, a GCE image is created from the GCE disk and the
+GCE disk is deleted.
```

Comment:
How this description relates to the import user guide doc?  Also talking about disk deletion in this high level introduction might cause confusion.

Reply:
Modified the existing image import doc. Merged in this info.